Field.h

```diff
@@ -125,4 +126,38 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
+};
+
+class CharField: public Field {
+    
+    private:
+        string fieldValue;
+        int maxLength;
+        
+    public:
+        CharField(string name, int maxLen, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
+            Field(name, CHAR, primary, unique, nullable, maxLen) 
+        { 
+            fieldValue = defaultValue;
+            maxLength = maxLen;
+        } 
+        
+        void setValue(string value) {
+            setNull(false);
+            fieldValue = checkMaxLen(value, maxLength);
+        }
+        
+        string getValue() {
+            return fieldValue;
+        }
+        int getMaxLen() {
+            return maxLength;
+        }
+        
+        string checkMaxLen(string newVal, int MaxLenVal) {
```

Comment:
Actually, for consistency, this should be called `checkMaxLength` and the first parameter should be `newValue`. The last parameter is not required as it is already a class attribute.